src/main/java/ui/TestController.java

```diff
@@ -25,6 +25,11 @@
  * instance which can be called from tests that need to access the UI class directly.
  */
 public final class TestController {
+
+    public static final String TEST_DIRECTORY = "settings";
+    public static final String TEST_SESSION_CONFIG_FILENAME = "test.json";
+    public static final String TEST_USER_CONFIG_FILENAME = "test_user.json";
```

Comment:
Hmm, @damithc mentioned that we should separate test code from normal code, so I'm not putting them in the `Preferences` class. I'm not sure where else these test constants should be, `TestController` sounds like a reasonable place to have them, since it handles test stuff already.